xyz

```diff
@@ -119,8 +119,14 @@ next_version=$("$dir/node_modules/.bin/semver" -i "$increment" "$version") ||
 message="${message_template//X.Y.Z/$next_version}"
 tag="${tag_template//X.Y.Z/$next_version}"
 
-bold=$(tput bold)
-reset=$(tput sgr0)
+if [[ -t 1 ]] && [[ -n "$TERM" ]] && which tput &>/dev/null; then
+  bold=$(tput bold)
+  reset=$(tput sgr0)
+else
+  bold=""
+  reset=""
```

Comment:
> Won't bold and reset default to empty string if not defined?

Yes, however, when running `xyz` in gitbash, the script fails like so:

```sh
$ ./xyz --dry-run
./xyz: line 122: tput: command not found
```

With the guard above included, it works just fine:

```sh
$ ./xyz --dry-run
Current version is 0.5.0. Press [enter] to publish xyz@0.5.1.
npm prune
npm test
node -e 'var o = require("./package.json"); o.version = "0.5.1"; require("fs").writeFileSync("./package.json", JSON.stringify(o, null, 2) + "\n");'
git add 'package.json'
git commit --message 'Version 0.5.1'
git tag --annotate 'v0.5.1' --message 'Version 0.5.1'
git push 'origin' 'refs/heads/master' 'refs/tags/v0.5.1'
npm publish
```

But you don't get the nice bloding of the version numbers.